bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/SecurityCapability.java

```diff
@@ -47,7 +47,7 @@
 class SecurityCapability extends RestCapability<SecurityApi> {
     private final Logger logger = LoggerFactory.getLogger(SecurityCapability.class);
 
-    private static final Map<String, HomeEvent> eventBuffer = new HashMap<>();
+    private static final Map<String, HomeEvent> EVENT_BUFFER = new HashMap<>();
```

Comment:
@clinique : I have a BIG doubt if it must be static or not ?

Reply:
No, it should not be static.

Reply:
Ok, fixed.